### ISSUE TITLE
Add Batch Limit for Stackdriver Exporter

### ISSUE THREAD

**Issue by marvinkite** [GoogleCloudPlatform/opentelemetry-operations-go#575]
On high-velocity metrics, you get:

```
InvalidArgument - Field timeSeries had an invalid value: A maximum of 200 TimeSeries can be written in a single request.
```

Apparently Stackdriver backend has the limit that each request cannot batch more than 200 time series so exporter needs to do some processing.

I have this issue with f0e0c76a62e3 commit

```golang 
github.com/GoogleCloudPlatform/opentelemetry-operations-go/exporter/metric v0.34.3-0.20230106160205-f0e0c76a62e3
```

See GoogleCloudPlatform/opentelemetry-operations-js/issues/87

**Comment by dashpole** [comment]
Thanks for reporting this!  This is definitely something we need to implement.  It exists on the googlecloud collector exporter for metrics, but not on the SDK exporter right now.

**Comment by khongSC** [comment]
Would it be possible to get this ticket prioritized? I currently can't use the latest version of the exporter in production because it will constantly throw the maximum time series error. Thanks!

**Comment by dashpole** [comment]
@damemi go for it.

**Comment by gagata** [comment]
hi @damemi was there any progress on this?

**Comment by knightpp** [comment]
A naive implementation could be <https://github.com/GoogleCloudPlatform/opentelemetry-operations-go/compare/main...knightpp:opentelemetry-operations-go:issue-575>.

**Comment by zchee** [comment]
@knightpp cc: @dashpole 
Could you create new PR based above change? My company team has same issue.
If can't send PR with any reason, I'll create instead, based by your changes with `Co-Author` mark.

**Comment by knightpp** [comment]
@zchee I've created a PR. I hope the PR is good enough to be merged. Thanks.

**Comment by damemi** [comment]
Fixed by #617, thanks @knightpp for contributing that!
